lib/src/services/ds_bottom_sheet.service.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSBottomSheetService {
+  final BuildContext context;
+  final Widget child;
+
+  DSBottomSheetService({
+    required this.context,
+    required this.child,
+  });
+
+  Widget _buildBottomSheet({ScrollController? controller}) {
+    return Container(
+      decoration: const BoxDecoration(
```

Comment:
You should add a margin here to avoid the system status bar.

<img width="630" alt="image" src="https://user-images.githubusercontent.com/91232521/195172310-9f356600-7dc0-46b2-9c00-94165d94fbbd.png">

```dart
margin: EdgeInsets.only(
  top: MediaQueryData.fromWindow(window).padding.top + 10,
),
```

Reply:
done